lib/desktop_screens_new/history_screen/history_desktop.dart

```diff
@@ -64,8 +64,6 @@ class _HistoryDesktopScreenState extends State<HistoryDesktopScreen> {
               filteredFiles = getDisplayFileData(provider.allFilesHistory);
             },
             successBuilder: (provider) {
-              filteredFiles = getDisplayFileData(provider.allFilesHistory);
```

Comment:
Why is this removed ?
It will not refresh history screen on receiving new file.

Reply:
@sachins-geekyants If you put this code here, it will block search method